docs/content/contribute/how-to-contribute/_index.adoc

```diff
@@ -0,0 +1,29 @@
+
+---
+title: "How to contribute"
+weight: 2
+description: >
+  How to contribute the ODU/CGA CubeSat flight software
+---
+
+Contributions to the ODU/CGA CubeSat flight software can come in a variety of forms. This document identifies 3 forms of contribution... subject-matter expertise, technical writing, and code.
```

Comment:
Replace "forms" with "roles".

Replace "..." with "which are"

Define subject-matter expert **(SME)** here.  Replace all other instances of "subject-matter expert" with "SME" except for the heading just below.

Replace "technical writing" with "technical writer" and "code" with "developer"

Reply:
resolved as per https://github.com/ODU-CGA-CubeSat/mission-conops/pull/28/commits/a8fd7480401dbe323c483ba99755407f58910fcd